app/com/gu/floodgate/AppComponents.scala

```diff
@@ -66,11 +68,16 @@ class AppComponents(context: Context) extends BuiltInComponentsFromContext(conte
   val runningJobController = new RunningJobApi(runningJobService)
   val jobHistoryController = new JobHistoryApi(jobHistoryService)
 
-  val appController = new Application(wsClient, configuration)
+  val clientId = configuration.get[String]("google.clientid")
+  val clientSecret = configuration.get[String]("google.clientsecret")
+  val redirectUrl = configuration.get[String]("google.oauthcallback")
+  val domain = "guardian.co.uk"
```

Comment:
Previously it was in the AuthAction trait, they've been moved to AppComponents following the conventions in `play-googleauth`. See here: https://github.com/guardian/play-googleauth/blob/master/example/app/AppComponents.scala

I imagine its for ease of use to be passed to various controllers.